splunk/resource_splunk_configs_conf_test.go

```diff
@@ -9,8 +9,8 @@ import (
 )
 
 const newConfigsConf = `
-resource "splunk_configs_conf" "tftest-stanza" {
-	name = "tftest/tftest_stanza"
+resource "splunk_configs_conf" "tf_test-stanza" {
+	name = "tf_test/tftest_stanza"
```

Comment:
There's no reason to change the resource name. Doing so caused the tests to fail, though I see you updated line 55.  Line 32 also needed to be updated, and really line 55 should reference `resourceName`, but that's not your doing.